docs/integrations/builtin/core-nodes/n8n-nodes-base.merge.md

```diff
@@ -7,96 +7,86 @@ contentType: integration
 
-#### Merge by fields
+#### Matching Fields
 
-Compare items by field values. Enter the fields you want to compare in **Fields to Match**.
+Compare items by field values. Enter the fields you want to compare in **Fields to Match**. 
 
 n8n's default behavior is to keep matching items. You can change this using the **Output Type** setting:
 
 * **Keep Matches**: Merge items that match. This is like an inner join.
 * **Keep Non-Matches**: Merge items that don't match.
 * **Keep Everything**: Merge items together that do match and include items that don't match. This is like an outer join.
-* **Enrich Input 1**: keep all data from Input 1, and add matching data from Input 2. This is like a left join.
-* **Enrich Input 2**: keep all data from Input 2, and add matching data from Input 1. This is like a right join.
+* **Enrich Input 1**: Keep all data from Input 1, and add matching data from Input 2. This is like a left join.
+* **Enrich Input 2**: Keep all data from Input 2, and add matching data from Input 1. This is like a right join.
 
 ![Diagram](/_images/integrations/builtin/core-nodes/merge/merge-by-field-diagram.png)
 
 
-##### Field value clashes
-
---8<-- "_snippets/integrations/builtin/core-nodes/merge/field-value-clash.md"
-
-##### Multiple matches
-
-Matching by field can generate multiple matches if the inputs contain duplicate data. To handle this, select **Add Option > Multiple Matches**. Then choose:
-
-* **Include All Matches**: Output multiple items if there are multiple matches, one for each match.
-* **Include First Match Only**: Keep the first item per match and discard the remaining multiple matches.
-
-
-#### Merge by position
+#### Position
 
 Combine items based on their order. The item at index 0 in Input 1 merges with the item at index 0 in Input 2, and so on.
 
 ![Diagram](/_images/integrations/builtin/core-nodes/merge/merge-by-position-diagram.png)
 
-##### Inputs with different numbers of items
-
-If there are more items in one input than the other, the default behavior is to leave out the items without a match. Choose **Add Option** > **Include Any Unpaired Items** to keep the unmatched items.
-
-##### Field value clashes
-
---8<-- "_snippets/integrations/builtin/core-nodes/merge/field-value-clash.md"
-
-#### Multiplex
+#### All Possible Combinations
 
 Output all possible item combinations, while merging fields with the same name.
 
 ![Diagram](/_images/integrations/builtin/core-nodes/merge/multiplex-diagram.png)
 
-##### Field value clashes
-
---8<-- "_snippets/integrations/builtin/core-nodes/merge/field-value-clash.md"
+#### Combine mode options
 
-#### Options
+When merging data by **Mode > Combine**, you can set these **Options**:
 
-When combining branches, you can set **Options**:
+* **Clash Handling**: Choose how to merge when branches clash, or when there are sub-fields. Refer to [below](/integrations/builtin/core-nodes/n8n-nodes-base.merge/#combine-mode-options/) for details.
```

Comment:
```suggestion
* **Clash Handling**: Choose how to merge when branches clash, or when there are sub-fields. Refer to [Clash handling](#clash-handling) for details.
```

I **think** this is where you were trying to send folks? Mkdocs is smart enough to handle anchors, so you should be able to just reference the subheading slug here.

Reply:
yes, that's the link I intended, thanks for checking 🙇‍♀️